evaluator/evaluator_test.go

```diff
@@ -508,7 +509,10 @@ b, c = [1, 2]; b`, 1},
 		a.int()
 				`, 10},
 	}
-
+	if runtime.GOOS == "windows" {
```

Comment:
I'd be worried not testing all of this stuff on windows. Maybe we can do the opposite -- check if we're not running on windows and push the test case inside the `tests`?